### PR TITLE
[ReadMe]: Update parent-child resource conditional description & updated format

### PR DESCRIPTION
**Pull request by MrMCake**
# Description

- Update parent-child resource conditional description & updated format

# Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update (Wiki)

### CODE REVIEW

File: constructs/Microsoft.Management/managementGroup-structure/readme.md [hunk 1]
```diff
@@ -123,5 +123,3 @@ This template is using a **Tenant level deployment**, meaning the user/principal
 
 ## Additional resources
```
Comment (eriqua) [inline constructs/Microsoft.Management/managementGroup-structure/readme.md:124]: Let's remove this line

File: arm/Microsoft.KubernetesConfiguration/extensions/deploy.bicep [hunk 2]
```diff
@@ -1,10 +1,10 @@
-@description('Required. The name of the Flux Configuration')
+@description('Required. The name of the Flux Configuration.')
 param name string
 
 @description('Optional. Enable telemetry via the Customer Usage Attribution ID (GUID).')
 param enableDefaultTelemetry bool = true
 
-@description('Required. The name of the AKS cluster that should be configured.')
+@description('Conditional. The name of the AKS cluster that should be configured. Required if the template is used in a standalone deployment.')
```
Comment (eriqua) [inline arm/Microsoft.KubernetesConfiguration/extensions/deploy.bicep:7]: This is a separate provider so technically not an AKS child resource. This change should probably be rolled back.
```suggestion
@description('Required. The name of the AKS cluster that should be configured.')
```

Comment (MrMCake) [reply arm/Microsoft.KubernetesConfiguration/extensions/deploy.bicep]: done

**Review by eriqua — APPROVED** [review]
🆗

**Pull request merged**